pkg/lib/cockpit-components-logs-panel.jsx

```diff
@@ -50,12 +50,10 @@ class JournalOutput {
                 <div className="cockpit-log-warning" role="cell">
                     { warning
                         ? <i className="fa fa-exclamation-triangle" />
-                        : null
-                    }
+                        : null}
```

Comment:
This is another debatable one -- now it's harder to see the corresponding opening bracket. Maybe let's revert/disable that one as well for now?